pkgs/development/libraries/opensubdiv/default.nix

```diff
@@ -58,10 +60,11 @@ stdenv.mkDerivation rec {
     [ "-DNO_TUTORIALS=1"
       "-DNO_REGRESSION=1"
       "-DNO_EXAMPLES=1"
+      "-DNO_DX=1"
       (lib.cmakeBool "NO_METAL" (!stdenv.isDarwin))
       (lib.cmakeBool "NO_OPENCL" (!openclSupport))
       (lib.cmakeBool "NO_CUDA" (!cudaSupport))
-    ] ++ lib.optionals (!stdenv.isDarwin) [
+    ] ++ lib.optionals stdenv.isLinux [
```

Comment:
Maybe the previous person meant `isUnix && ! isDarwin` here.

Reply:
I'm happy to make that change, but I'm not sure if it will make things build on non-linux (& non-darwin) unixes.

Reply:
Done